swc-uxp-starter/package.json

```diff
@@ -29,6 +29,9 @@
   "resolutions": {
     "@spectrum-web-components/theme": "0.14.9",
     "@spectrum-web-components/banner": "0.9.2",
-    "@spectrum-web-components/link": "0.14.1"
+    "@spectrum-web-components/link": "0.14.1",
+    "@spectrum-web-components/shared": "0.15.4",
+    "@spectrum-web-components/styles": "0.22.1",
+    "@spectrum-web-components/base": "0.7.3"
```

Comment:
Please add the explanation for this in the README

Reply:
The components added (shared, styles, base) are not UI components but are required to be in the resolution block just as the `theme` for version consistency. As we have not mentioned the theme inclusion in the resolution block in the README, I have updated the screenshot with these umbrella components added.

Reply:
And these are going to be common to all components, right?

Reply:
Yes, they are common for all the components